src/build/dotnet/jobs/nuget-package.job.yaml

```diff
@@ -1,26 +1,110 @@
+# Job to build a NuGet Package that will be hosted on an internal VSTS feed
 parameters:
-  projects: '**/*.csproj'
-  tests: '**/*Tests/*.csproj'
-  displayName: 'Build Package'
-  skipDependencyChecks: false
-  signAssemblies: true
-  releaseBranchName: 'master'
-  packageSource: 'internalPublic'
+  - name: displayName
+    default: 'Build Package'
+  - name: projects
+    default: '**/*.csproj'
+  - name: tests
+    default: '**/*Tests/*.csproj'
+  - name: skipDependencyChecks
+    default: false
+  - name: signAssemblies
+    default: true
+  - name: releaseBranchName
+    default: 'master,main'
+  - name: shouldRunTests
+    default: true
+  - name: applyVersioning
+    default: true
+  - name: artifactPath
+    default: ''
+  - name: artifactName
```

Comment:
is there any reason we parameterise this here and not in `signed-nuget-package.job`?

Reply:
Have added to signed job